client/src/App.js

```diff
@@ -2,6 +2,8 @@ import SplashPage from "./components/Page/SplashPage";
 import AppHeader from "./components/AppHeader/AppHeader";
 import GlobalStyle from "./GlobalStyle";
 import { Buttons } from "./components/Button/Button";
+import React, { useState } from "react";
```

Comment:
do you need useState here?

Reply:
No. Not necessary. I will delete. Thanks :)